xml/core_registers.xml

```diff
@@ -70,7 +70,48 @@ same project unless stated otherwise.
             available version of this spec.
             </value>
         </field>
-        <field name="0" bits="27:18" access="R" reset="0" />
+        <field name="0" bits="27" access="R" reset="0" />
+        <field name="extcause" bits="26:24" access="R" reset="0">
+            When {dcsr-cause} is 7, this optional field contains the value of a
+            more specific halt reason than "other." Otherwise it contains 0.
+
+            <value v="0" name="dbltrp">
+                An exception or interrupt occurred while the trap handler was in
```

Comment:
```suggestion
                The hart entered a critical error state.  Part of the ((Smdbltrp)) extension.
```
Reworded to:
1. Use the critical error term to tie it back to the `cetrig`
2. Such an exception/interrupt may not always lead to a critical error (e.g., at S-mode or VS-mode it does not) and in M-mode it does not if it was not the RNMI handler and RNMI was implemented. So this would avoid needing to put all conditions in here.

Reply:
OK.